### PR TITLE
Web API: PerformanceNavigation - Correct to Show as Unsupported in Firefox

### PR DESCRIPTION
**Pull request by Kelderic**
Firefox does not support this. Please see: https://bugzilla.mozilla.org/show_bug.cgi?id=1459711

### CODE REVIEW

File: api/PerformanceNavigation.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
             "version_added": true
           },
           "firefox": {
-            "version_added": "7"
+            "version_added": false
```
Comment (ddbeck) [inline api/PerformanceNavigation.json]: I'd undo this change

File: api/PerformanceNavigation.json [hunk 2]
```diff
@@ -17,10 +17,10 @@
...
           },
           "firefox_android": {
-            "version_added": "7"
+            "version_added": false
```
Comment (ddbeck) [inline api/PerformanceNavigation.json]: I'd undo this change

File: api/PerformanceNavigation.json [hunk 3]
```diff
@@ -118,10 +118,10 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": "7"
+              "version_added": false
```
Comment (ddbeck) [inline api/PerformanceNavigation.json]: Based on the bug report (and playing with the API myself), I'm not sure that `redirectCount` matters at all. I'd undo this change

File: api/PerformanceNavigation.json [hunk 4]
```diff
@@ -118,10 +118,10 @@
...
             },
             "firefox_android": {
-              "version_added": "7"
+              "version_added": false
```
Comment (ddbeck) [inline api/PerformanceNavigation.json]: Like before, I'd undo this change

File: api/PerformanceNavigation.json [hunk 5]
```diff
@@ -67,10 +67,10 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": "7"
+              "version_added": false
```
Comment (ddbeck) [inline api/PerformanceNavigation.json]: Instead of `false` here, I'd add a note like this:

```suggestion
              "version_added": "7",
              "notes": "This property can return incorrect values. See <a href='https://bugzil.la/1459711'>bug 1459711</a>."
```

What do you think?

File: api/PerformanceNavigation.json [hunk 6]
```diff
@@ -169,10 +169,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": "25"
+              "version_added": false
```
Comment (ddbeck) [inline api/PerformanceNavigation.json]: I'm suggesting that we do something similar to the change for `type` here:

```suggestion
              "version_added": "25",
              "notes": "This property can return incorrect <code>type</code> values. See <a href='https://bugzil.la/1459711'>bug 1459711</a>."
```

File: api/PerformanceNavigation.json [hunk 7]
```diff
@@ -169,10 +169,10 @@
...
             },
             "firefox_android": {
-              "version_added": "25"
+              "version_added": false
```
Comment (ddbeck) [inline api/PerformanceNavigation.json]: Like before:

```suggestion
              "version_added": "25",
              "notes": "This property can return incorrect <code>type</code> values. See <a href='https://bugzil.la/1459711'>bug 1459711</a>."
```

**Comment by ddbeck** [comment]
Hi @Kelderic, to make the suggested changes you can make a new commit on your branch, then push that to your fork on GitHub. For the inline suggestions I left, you can also use the GitHub UI to apply those changes specifically: https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change

**Comment by Kelderic** [comment]
@Elchi3 Oops! I'm sorry; life got really busy this past month, and I kinda forgot about this. Thanks for finishing it up.
